blockbase/functions.php

```diff
@@ -57,8 +57,8 @@ function( $settings ) {
 		);
 
 	}
-	add_action( 'after_setup_theme', 'blockbase_support', 9 );
 endif;
+add_action( 'after_setup_theme', 'blockbase_support', 9 );
```

Comment:
I'm not sure we want to do this. The way I see it, if that function is defined elsewhere, its probably by some totally unrelated code, in which case we probably want to leave it to run itself...

Reply:
Ah, okay? I assumed it to be a "pluggable" function for child themes, like many other Automattic themes use ;)

Reply:
Oh yes, I see what you mean. 👍